### DISCUSSION TITLE
[ SkinnedMesh ] Way to find out bone index number which is refered by skinIndex ?

### DISCUSSION THREAD

**Discussion by Duke-2** [pixiv/three-vrm#1222] [Help]
# Background
- I would like to find out the list of mesh vertices which is bind to 'head' bone.
- The vertices has bone information the as index number in ` .geometry.attributes.skinIndex.array`
- I calculated the index number of the bone as the alphabetical order of `humanoid.humanBones`  's children nodes
- However, the vertices list seems not correct because it is not axis symmetric
# Question
1. Way to find out bone index number which is refered by skinIndex ?
2. Any easier way to get all the vertices bind to 'head' bone?

**Comment by ke456-png** [comment]
How will it be put to use?  [VRMFirstPerson](https://pixiv.github.io/three-vrm/packages/three-vrm/docs/classes/VRMFirstPerson.html) might be helpful.
